project/src/components/cities/cities.tsx

```diff
@@ -0,0 +1,33 @@
+import { Link, useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import cities from './../../mocks/cities';
+
+function Cities() {
+
+  function getLinkClassName(city:string, activeCity:string|null):string {
```

Comment:
```suggestion
  function getLinkClassName(city: string, activeCity: string | null): string {
```

Пожалуйста: не пиши слитно. Очень больно читать